common/test/acceptance/pages/studio/container.py

```diff
@@ -191,10 +193,19 @@ def toggle_staff_lock(self, inherits_staff_lock=False):
         Returns True if the lock is now enabled, else False.
         """
         was_locked_initially = self.is_staff_locked
+
+        # Sections and Units lock differently than subsections, so try both actions
```

Comment:
Is it possible to be explicit about this by checking which modal is open (for subsection or not) and then clicking the exactly correct UI?